cypress/test/layout/GenericLayout.cy.js

```diff
@@ -0,0 +1,20 @@
+import MockRouter from '../../utils/router';
+import GenericLayout from '../../../components/layout/GenericLayout';
+describe('GenericLayout Component', () => {
+  it('renders correct GenericLayout component', () => {
+    cy.mount(
+      <MockRouter asPath="/">
+        {' '}
+        <GenericLayout
+          title="Test Title"
+          description="Test Description"
+          image="test-image.jpg"
+          wide={false}
+          hideBanner={true}
```

Comment:
Make separate cases for these properties inside component.

Reply:
done